test/unit-tests/fiber.mlt

```diff
@@ -0,0 +1,168 @@
+open Dune;;
+open Fiber;;
+open Fiber.O;;
+
+let opt_value_exn = function
+  | Some x -> x
+  | None -> failwith "option is empty"
+;;
```

Comment:
Why not just use stdune? `open Stdune` and `Option.value_exn`. You'd also have to add the object files of the library, but that's not so bad.